### PR TITLE
Salt key and campaign dashboard view amendments

### PR DESCRIPTION
**Pull request by sarmaevanidfe**

### CODE REVIEW

**Review by arvinthseran — APPROVED** [review]
Functioanal testing results can be found under https://skillsfundingagency.atlassian.net/browse/ASINTEL-691

File: src/SFA.DAS.DATA.MANAGEMENT/SFA.DAS.Data.Management.Database.sqlproj.user [hunk 1]
```diff
@@ -0,0 +1,3 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+</Project>
```
Comment (arvinthseran) [inline src/SFA.DAS.DATA.MANAGEMENT/SFA.DAS.Data.Management.Database.sqlproj.user:3]: this looks like user profile file, may not be necessary in this PR. can you double check please.

Comment (uhimabindu) [reply src/SFA.DAS.DATA.MANAGEMENT/SFA.DAS.Data.Management.Database.sqlproj.user:3]: Yes, This is not required. Sarma, Can you remove this please. Thank You.

**Review by uhimabindu — CHANGES_REQUESTED** [review]
SFA.DAS.Data.Management.Database.sqlprojAssemblyReference.cache File is not required. Please Remove it.

**Review by uhimabindu — CHANGES_REQUESTED** [review]
Can you also please resolve merge conflicts by taking new changes from Master.

**Pull request closed without merging**